.NET/Microsoft.Recognizers.Text.DateTime/Dutch/Extractors/DutchDateExtractorConfiguration.cs

```diff
@@ -69,6 +69,9 @@ public class DutchDateExtractorConfiguration : BaseOptionsConfiguration, IDateEx
         public static readonly Regex RelativeMonthRegex =
             new Regex(DateTimeDefinitions.RelativeMonthRegex, RegexOptions.Singleline);
 
+        public static readonly Regex RelativeRegex =
+            new Regex(DateTimeDefinitions.RelativeRegex, RegexOptions.Singleline);
+
         public static readonly Regex PrefixArticleRegex =
```

Comment:
DateExtractor2 should also be removed from every other language and their YAML files, right?

Reply:
Yes, but I found that we can only remove it from English and Dutch, while in other languages it is defined differently than in English.

Reply:
You mean that in some cultures regex 1 and 2 are not both cases of the same regex with a different ending?